webui/src/App.js

```diff
@@ -456,16 +464,14 @@ function Plot(props) {
       previousTimestamp = timestamp;
 
       // Add background fill.
-      let grd = ctx.createLinearGradient(canvas.width/2, 0, canvas.width/2 ,canvas.height);
-      grd.addColorStop(0, 'white');
-      grd.addColorStop(1, 'lightgray');
-      ctx.fillStyle = grd;
+      ctx.fillStyle = "#f8f9fa";
```

Comment:
I removed the gradient for more uniform contrast against the plot lines, but I could put it back. The color is the same as the navbar.